nemo/collections/tts/modules/audio_codec_modules.py

```diff
@@ -402,7 +432,7 @@ def decode(self, indices: torch.Tensor, input_len: Optional[torch.Tensor] = None
         return dequantized
 
 
-class GroupFiniteScalarQuantizer(NeuralModule):
+class GroupFiniteScalarQuantizer(VectorQuantizerBase):
```

Comment:
If the quantizers have the same interface then would it make sense to have just 1 grouped quantizer implementation/interface? I guess there would need to be a small amount of logic to specify how the underlying codebooks are created.